jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -115,10 +120,6 @@
   static final String STATEMENT_PRECODE_KEY_TEMPLATE = "%s.statementPrecode";
   static final String DOT = ".";
 
-  private static final char WHITESPACE = ' ';
-  private static final char NEWLINE = '\n';
-  private static final char TAB = '\t';
-  private static final String TABLE_MAGIC_TAG = "%table ";
```

Comment:
why is this not needed?

Reply:
It has been moved to JDBCUtils

Reply:
I don't see the `TABLE_MAGIC_TAG = "%table ";`?

Reply:
It has been used only for knowing type in writer